MantidQt/SliceViewer/inc/MantidQtSliceViewer/PeakViewColor.h

```diff
@@ -0,0 +1,28 @@
+#ifndef MANTID_SLICEVIEWER_PEAK_VIEW_COLOR_H_
+#define MANTID_SLICEVIEWER_PEAK_VIEW_COLOR_H_
+
+#include <QColor>
+
```

Comment:
Could use some explanation as to what this type is. `colorCross`, `colorSphere` etc, are publicly accessible. Is that desired?